examples/in_op_sytanx.conf

```diff
@@ -0,0 +1,13 @@
+[request_definition]
```

Comment:
Use Go's example

Reply:
``aviator``  does not seem to support  ``(a,b)`` as a array like Go example.In fact,``in`` can only support pattten like ``r.sub.name in (r.obj.admins)`` becase the regexp is implemented like that.It's not flexible like Go-casbin

Reply:
Add the example to docs

Reply:
the example``m = r.sub.name == p.sub && r.sub.name in (r.obj.admins)`` is same as docs example ``r.sub.Name in (r.obj.Admins)``.is it not necessary?

Reply:
Then not needed.